web/src/components/DatePickerWrapper.js

```diff
@@ -0,0 +1,73 @@
+import omit from 'lodash.omit';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { DateRangePicker } from 'react-dates';
+
+const dateToStr = (date, fmt = 'YYYY-MM-DD') => date.format(fmt);
+const strToDate = str => (str ? moment(str) : null);
+
+class DatePickerWrapper extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      focusedInput: null,
+      startDate: strToDate(props.initialStartDate),
+      endDate: strToDate(props.initialEndDate)
+    };
+  }
+
+  onDatesChange = ({ startDate, endDate }) => {
+    if (this.props.onChange) {
+      this.props.onChange({
+        start: startDate && dateToStr(startDate),
+        end: endDate && dateToStr(endDate)
+      });
+    }
+
+    this.setState({ startDate, endDate });
+  };
+
+  onFocusChange = focusedInput => {
+    this.setState({ focusedInput });
+  };
+
+  render() {
+    const { focusedInput, startDate, endDate } = this.state;
+
+    const props = omit(this.props, [
```

Comment:
Does the DateRangePicker complain if you send it props it doesn't know about?  I assume that's what this is for.

Reply:
yes!